README.md

```diff
@@ -16,6 +16,7 @@ wgpu-rs can target both the natively supported backends and WASM directly.
 
 ![Cube](etc/example-cube.png) ![Shadow](etc/example-shadow.png) ![MipMap](etc/example-mipmap.png) ![Skybox](etc/example-skybox.gif)
 ![vange-rs](etc/vange-rs.png) ![Brawl](etc/brawl-attack.gif) ![GLX map](etc/glx-map.png) ![Harmony](etc/harmony-rs.jpg)
+![Blub](etc/blub-fluid-small.gif)
```

Comment:
To keep it tidy on the front page for desktop, why not place this between the vange-rs example and the brawl example?

Reply:
tried, actually, didn't look much better though. Also github already has a new design on the way (you can opt into it) which moves everything around again _sigh_

Reply:
ah actually. I figure it does look a bit better. updated.